package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@sharekey/meteor-desktop",
-  "version": "2.2.14",
+  "version": "2.2.15",
```

Comment:
pls remove from this pr
I will bump version by myself just before release new version